worlds/generic/docs/setup_en.md

```diff
@@ -136,6 +136,24 @@ are rolling locally, ensure this file is edited to your liking **before** rollin
 when running the Archipelago Installation software. If you have changed settings in this file, and would like to retain 
 them, you may rename the file to `options.yaml`. 
 
+### Adding new worlds
```

Comment:
This should probably use the word "custom" instead of "new", or maybe a different wording altogether

In general, I think it's important to do a pass on this PR for disambiguation between "adding a world" = "developing a new .apworld" vs "adding a world" = "installing an existing custom world into your local ap install", similar / opposite to what https://github.com/ArchipelagoMW/Archipelago/pull/5457 is trying to achieve